yhondanilo/chess.rb

```diff
@@ -0,0 +1,41 @@
+class Chess
+
+	attr_accessor :fila, :columna
+
+	def initialize(fila, columna)
+		@fila, @columna=fila, columna
+		intercalar
+	end
+
+	def intercalar
+		@board=Array.new(@fila){Array.new(@columna)}
+		@board[0][0]=1
+		for f in 0..self.fila-1
+		 for c in 0..self.columna-1
+		  if @board[f][c-1]==0
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Surrounding space missing for operator '=='.<br>Surrounding space missing for operator '-'.<br>Tab detected.